### PR TITLE
feat: show podcasting info in transactions

### LINKED ISSUE #16: Display custom records and podcasting data
This should be available on some transactions in metadata["custom_records"]. See how the Alby Browser Extension shows podcasting info in transactions for reference.

### PR DESCRIPTION
**Pull request by im-adithya**
## Description
Fixes #16

This adds boostagram tlv record info only. We don't show other info in the Hub, is it worth showing the other custom record information here?

## Screenshots
<table>
<tr>
<td>

</td>
<td>

</td>

</tr>
</table>

### CODE REVIEW

**Comment by reneaaron** [comment]
Can we get rid of the seemingly duplicate "description" & "message" here? Or could they be different?

**Comment by im-adithya** [comment]
I think we should change this in the Hub, if the transaction already has the tlv records/boostagram, then it doesn't make sense to change the description of the transaction to the one fetched from tlv records right?

Similarly we do this during zap requests made in the rails app, we change the description to the content from the zap... (this is should also be changed I feel)

File: pages/Transaction.tsx [hunk 1]
```diff
@@ -2,90 +2,129 @@ import { Nip47Transaction } from "@getalby/sdk/dist/NWCClient";
...
+  value_msat_total: number;
+}
 
 export function Transaction() {
   const { transactionJSON } = useLocalSearchParams() as unknown as {
     transactionJSON: string;
   };
   const transaction: Nip47Transaction = JSON.parse(transactionJSON);
   const getFiatAmount = useGetFiatAmount();
+  const rawMetadata = transaction.metadata;
```
Comment (reneaaron) [inline pages/Transaction.tsx:43]: This part could potentially be memoized. Otherwise the records are parsed with every render...

**Review by reneaaron — APPROVED** [review]
One little nitpick, otherwise :+1: 

tACK

**Pull request merged**